sql/src/test/java/org/apache/druid/sql/calcite/CalciteArraysQueryTest.java

```diff
@@ -3159,34 +3273,26 @@ public void testUnnestWithFiltersInsideAndOutside()
     skipVectorize();
     testQuery(
         "SELECT d3 FROM\n"
-        + "  (select * from druid.numfoo where dim2='a') t,\n"
+        + "  (select * from druid.numfoo where dim2='a'),\n"
         + "  UNNEST(MV_TO_ARRAY(dim3)) as unnested (d3)\n"
-        + "WHERE t.dim1 <> 'foo'\n"
-        + "AND unnested.d3 <> 'b'",
+        + "WHERE dim1 <> 'foo'\n"
+        + "AND (unnested.d3 IN ('a', 'c') OR unnested.d3 LIKE '_')",
```

Comment:
what is the reason for changing this test instead of adding a new one?

Reply:
Will add new one